zavod/zavod/settings.py

```diff
@@ -116,3 +116,7 @@ def env_str(name: str, default: str) -> str:
 ZYTE_API_KEY = env.get("OPENSANCTIONS_ZYTE_API_KEY", None)
 OPENAI_API_KEY = env.get("OPENSANCTIONS_OPENAI_API_KEY", None)
 AZURE_OPENAI_ENDPOINT = env.get("OPENSANCTIONS_AZURE_OPENAI_ENDPOINT", None)
+
+ENABLE_SENTRY = env.get("ZAVOD_ENABLE_SENTRY", False)
```

Comment:
```suggestion
ENABLE_SENTRY = as_bool(env.get("ZAVOD_ENABLE_SENTRY", False))
```

So that "false" isn't True